### DISCUSSION TITLE
Bus Depot [Release]

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#323] [Map Releases]
## This is a manual migration of forum user Manuel Valle7's map release `BusDepot`. Below contains the original post content. You can view the original post [here](https://nzportable.forumotion.com/t2151-bus-depot-release).

Hello guys, it's been ages since I've been active but hey now that I'm out of school I actually have some free time. So yeah, hello everyone. Anyways, I've started working on a map recently, the one from Black Ops 2. 
For some reason the zombies are going to the wall, I don't know if that's part of the NZP demo but I'll have to fix that, hopefully with waypoints. I don't have any right now so if there are any volunteers, step up. The textures are crappy too so if anyone wants to compile a pack for me that would be of great use, I'm personally too lazy to do it.

It is considered as a W.I.P. so don't expect it to have any functionality whatsoever. I started it today so I didn't get very far in it. I know Senior did the same map for 1.1 but this works for the Demo. Anyways, tell me what you think and how I should improve. Oh and if you guys have found any bugs (probably by the chairs because I forgot to put an invisible block) tell me. 

__Screenshots:__
![0](https://github.com/nzp-team/nzportable/assets/26030250/e35a9e64-1aa9-4825-bb91-22097f3b106b)
![1](https://github.com/nzp-team/nzportable/assets/26030250/28f1ee14-5046-4529-b311-74c328686488)

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/12147908/BusDepot.zip)

**Comment by spectrum1-wave** [comment]
are there perks and pap, or vanilla bus depot?
